worker/provisioner/lxd-broker.go

```diff
@@ -194,6 +199,11 @@ func (broker *lxdBroker) LXDProfileNames(containerName string) ([]string, error)
 	return nameRetriever.LXDProfileNames(containerName)
 }
 
+// LXDProfileNames implements environs.LXDProfiler.
+//func (env *environ) LXDProfileNames(containerName string) ([]string, error) {
+//	return env.server.GetContainerProfiles(containerName)
+//}
```

Comment:
Why is this commented out, plan to add or remove this later?

Reply:
note to myself, removed.